client/src/components/FlightDetails.vue

```diff
@@ -296,6 +306,24 @@ const onReloadMetar = async () => {
     isFetchingMetar.value = false;
   }
 };
+// RECLAIM AIRSPACE VIOLATION
+const onClaimAirspaceViolation = async () => {
+  try {
+    if (!flight.value?.id) throw Error("No flight id");
+    const res = await ApiService.changeFlightProps(flight.value.id, {
+      flightStatus: FLIGHT_STATUS.IN_REVIEW,
```

Comment:
Ist das nicht etwas zu viel Kontrolle die man über den Endpunkt hat?
So kann man ja alles reinschreiben was man möchte. Ist jetzt als admin Funktion vermutlich kein großes Ding, aber ein Endpunkt für diese Funktion mit lediglich der FlightId als Parameter wäre eigentlich sauberer, oder?

Reply:
In dem Fall werden/müssen ja sogar direkt 2 Properties angeoasst werden "flightStatus" und "airspaceViolation".

Über den Endpoint hat man natürlich schon eine große Macht. Aber evtl brauchen wir ja in der Zukunft noch andere Änderungen. Dann wachsen wir bei den Endpoints noch mehr auf,